client-v2/src/components/FrontsEdit/CollectionComponents/Sublinks.tsx

```diff
@@ -12,15 +12,13 @@ import { collectionDropTypeBlacklist } from 'constants/fronts';
 
 const SublinkCollectionItemBody = styled(CollectionItemBody)<{
   dragHoverActive: boolean;
-  isClipboard: boolean;
 }>`
   display: flex;
   min-height: 30px;
-  border-top: ${({ isClipboard }) =>
-    isClipboard ? 'none' : '1px solid #c9c9c9'};
-  background-color: ${({ isClipboard, dragHoverActive }) =>
-    dragHoverActive ? `#ededed` : isClipboard ? '#f6f6f6' : '#fff'};
-  flex-direction: ${({ isClipboard }) => (isClipboard ? 'column' : 'row')};
+  border-top: 1px solid #c9c9c9;
+  background-color: ${({ dragHoverActive }) =>
+    dragHoverActive ? `#ededed` : '#fff'};
```

Comment:
In other places, we reference the theme.ts constants declared for these colours. Is there a reason we don't do it here?

Reply:
No, we should use the theme 👍 I'll make the change.